data/json/recipes/recipe_electronics.json

```diff
@@ -1033,6 +1033,28 @@
     [ [ "nail", 20 ] ],
     [ [ "pipe", 4 ] ]
   ]
+},{
+  "type" : "recipe",
+  "result": "xl_wind_turbine",
+  "category": "CC_ELECTRONIC",
+  "subcategory": "CSC_ELECTRONIC_PARTS",
+  "skill_used": "fabrication",
+  "skills_required": ["electronics", 5],
+  "difficulty": 5,
+  "time": 180000,
+  "reversible": true,
+  "decomp_learn": 2,
+  "autolearn": true,
+  "book_learn": [["manual_electronics", 7] , ["textbook_electronics", 6] , ["advanced_electronics", 6]],
+  "using": [ [ "welding_standard", 50 ] ],
+  "qualities" : [ {"id": "SCREW","level":1}, { "id": "HAMMER", "level": 2 }, { "id": "SAW_M", "level": 2 } ],
+  "components": [
+    [ [ "motor_small", 4 ], [ "alternator_bicycle", 4 ], [ "alternator_motorbike", 4 ] ],
```

Comment:
Any reason not to allow bigger alternators here?

Reply:
fair point. I'll get on that, bigger turbine would mean 4 small components or 1 big components.